src/SFA.DAS.Commitments.Database/StoredProcedures/StartATransferRequest.sql

```diff
@@ -0,0 +1,28 @@
+﻿CREATE PROCEDURE [dbo].[StartATransferRequest]
+	@commitmentid BIGINT,
+	@cost MONEY,
+	@trainingCourses NVARCHAR(MAX),
+	@transferRequestId BIGINT = null OUT
+AS
+BEGIN
+
+	DECLARE @OldApprovalStatus AS TINYINT = NULL
+
+	INSERT INTO [dbo].[TransferRequest] (CommitmentId, Cost, TrainingCourses, [Status]) VALUES (@commitmentid, @cost, @trainingCourses, 0)
+
+	SELECT @TransferRequestId = @@IDENTITY
```

Comment:
We don't have any triggers to trip over, but is it worth using scope_identity instead?